README.md

```diff
@@ -71,15 +71,17 @@ jobs:
 
 ### :moneybag: Available Inputs
 
-| Input Parm Name | Required | Default Value        | Description                                                                                         |
-| ---             | ---      | ---                  | ---                                                                                                 |
-| bundle          | false    | false                | If you want to use a version of a gem you maintain this is your best bet, longer time for load could be seen |
-| file_paths      | false    | 'app/**/*.html.haml' | Define the paths you wish to be linted per run. multiple paths can be on one line just add a space. |
-| version         | false    | latest GA            | Define a later version of haml_lint if latest is not needed                                         |
-| additional_gems | false    |                      | Additional Gems can be installed via one line with spaces and commands are supported like a version |
-| config_path     | false    | repo ./              | If the config path is another spot in the repo or not named .haml-lint.yml                          |
-| exclude_paths   | false    |                      | Define a list of paths to exclude from being linted.                                                |
-| fail_level      | false    | 'error'              | Can define 'error' or 'warning' to cause haml-lint to error out on                                  |
+Refer to the [official `haml-lint` documentation](https://github.com/sds/haml-lint#usage) for more information on the `haml-lint` options.
+
+| ****Input Parm Name**** | ****Required**** | ****Default Value**** | ****`haml-lint` option**** |                                                                                  ****Description****                                                                                  |              **Example**              |
```

Comment:
Whats the indication of haml-lint column, little confused by the usage/reference

Reply:
Was trying to indicate if it was a haml-lint config option. I can remove it unless you can think of a better way to word this. Basically my intent was to say 
> this is a haml-lint config option, refer to their documentation for more info or for how to use this.

Reply:
i think then instead of true false use yes/no